lib/Doctrine/ORM/Query.php

```diff
@@ -118,7 +118,7 @@ final class Query extends AbstractQuery
     /**
      * The current state of this query.
      *
-     * @var int
+     * @var self::STATE_*
```

Comment:
```suggestion
     * @var int
     * @psalm-var self::STATE_*
```

Let's keep literal types behind a vendor-prefix.